components/post-actions-bar/PostActionsBar.tsx

```diff
@@ -29,21 +34,54 @@ const PostActionsBar = ({
   creatorIdentifier,
 }: IProps) => {
   const router = useRouter();
-  const linkToCopy = `https://wwww.domain.con/posts/${identifier}`;
-  const { identifier: userIdentifier } = useUserInfo();
+  const [linkToCopy, setLinkToCopy] = useState<string>('');
+  const { identifier: userIdentifier, isLoggedIn } = useUserInfo();
+  const notify = () => {
+    toast(
+      <div className="bg-violet-200 p-4 rounded-lg flex flex-col text-center">
+        <div className="mb-4">
+          <Paragraph
+            size={EParagraphSizes.MEDIUM}
+            text="Please Login to like"
+          />
+        </div>
+        <ButtonMenu
+          name="login"
+          label="Login"
+          icon={EIConTypes.LOGOUT}
```

Comment:
shouldn't that be a login-icon?

Reply:
Perhaps we could extract it to toastify-component for login that can be used in several places.

Reply:
there is no login icon 😞. i changed to profile. or maybe we can add a login one.